todo/src/scripts/storageController.js

```diff
@@ -0,0 +1,88 @@
+class StorageController {
+    constructor(name) {
+        this.init();
+        this.data;
```

Comment:
What does it mean ? You do literally nothing here

Reply:
I rewrite ```constructor```
```suggestion
class StorageController {
    constructor() {
        if (!localStorage.getItem('toDo')){
            localStorage.setItem('toDo', JSON.stringify(
                {
                    'tasks': {
                        0: {
                            id: 0,
                            title: "Make cool App",
                            task: "Make id sort",
                            priority: "Low",
                            date: '11.00 01.01.2000'
                        }
                    },
                    'lastTaskId': 0,
                    'currentTasksId': [0],
                    'completeTasksId': [],
                }
            ));
        }
        this.data = JSON.parse(localStorage.getItem('toDo'));
    }
```